### PR TITLE
allows dehacked custom quit messages

### PR DESCRIPTION
**Pull request by T-117**
Allows setting custom quit messages though the normal dehacked processes.

### CODE REVIEW

**Comment by kraflab** [comment]
Do you have a link to a dehacked specification that includes this?

**Comment by T-117** [comment]
I don't know about specifications other than maybe this https://zdoom.org/wiki/Strings
but I noticed playing in Woof that the quit message strings were supported but not so in dsda.  I didn't know if that was intentional but if not I figured I'd add the feature here.

**Comment by kraflab** [comment]
Did you test it? It doesn't look like the dehacked strings are actually implemented.

File: prboom2/src/dstrings.c [hunk 1]
```diff
@@ -40,22 +40,22 @@ const char *const endmsg[]=
...
   // DOOM1
   QUITMSG,
-  "please don't leave, there's more\ndemons to toast!",
-  "let's beat it -- this is turning\ninto a bloodbath!",
-  "i wouldn't leave if i were you.\ndos is much worse.",
-  "you're trying to say you like dos\nbetter than me, right?",
-  "don't leave yet -- there's a\ndemon around that corner!",
-  "ya know, next time you come in here\ni'm gonna toast ya.",
-  "go ahead and leave. see if i care.",  // 1/15/98 killough
+  QUITMSG1,  // "please don't leave, there's more\ndemons to toast!",
```
Comment (fabiangreffrath) [inline prboom2/src/dstrings.c]: Shouldn't you use the `s_*` substitutes here instead?

Comment (T-117) [reply prboom2/src/dstrings.c]: good call.  Fixed.

**Comment by T-117** [comment]
@kraflab The most recent changes are tested and work as expected.

**Comment by kraflab** [comment]
👍

**Pull request merged**